src/Framework/ClassResolver/AbstractClassResolver.php

```diff
@@ -44,6 +45,14 @@ public function doResolve($caller): ?object
 
         $resolvedClassName = $this->findClassName($classInfo);
         if ($resolvedClassName === null) {
+            // Try once more with its parent class
```

Comment:
I think we should not try just once but recursively till there are no more parents.

Reply:
That's exactly what it does... maybe the comment it's misleading. I will improve it.